woocommerce/woocommerce-android
# Releasing customer more menu

### Description
This pull request introduces the new Customer section in the Menu Hub. This section now contains customers' contact information and includes some analytics, such as total spend and number of orders.

### Testing information

1. Open the app
2. Navigate to the Hub menu
3. Check that there is a new customer section
4. Tap on the customer section
5. Check that the list of customers is displayed
6. Check that the search bar is working as expected
7. Tap on a Customer
8. Check that the app navigates to the Customer details screen
9. Check that the customer information is correct.


### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/18119390/79ac3824-5241-47d6-b670-51af0720d68b
